addons/stock/stock.py

```diff
@@ -1589,6 +1589,8 @@ def rereserve_quants(self, cr, uid, picking, move_ids=[], context=None):
     def do_new_transfer(self, cr, uid, ids, context=None):
         pack_op_obj = self.pool['stock.pack.operation']
         data_obj = self.pool['ir.model.data']
+        if not context:
```

Comment:
why this one?

Reply:
@mart-e  because in return for the action it is using the context and if the context will be not passed from the call then it will have default value None. which will again cause the same problem because XML-RPC can't serialize None.

Reply:
I tried without setting this context and it works.  You can also tell xmlrpc to allow Nones.

Reply:
@jco-odoo  are you sure in your case it was returning the Action?
for "allow None" option I know in Python we can do it but In my knowledge not in every language. Especially from PHP.

Reply:
@jco-odoo could you please recheck? We really need this one merged in to fix an issue we have at a customer.